common/Labs.Head.Uwp.props

```diff
@@ -9,4 +9,8 @@
   <PropertyGroup>
     <TargetFramework>$(UwpTargetFramework)</TargetFramework>
   </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)' == 'Release'">
+    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+  </PropertyGroup>
```

Comment:
This should remain in the Uwp.Base.props file for the test projects being built in release mode as well.